### PR TITLE
Tests for Coulomb_RF and quartic

### LINKED ISSUE #2688: Test force/energy/pressure for coulomb_dh, coulomb_rf, quartic

### PR DESCRIPTION
**Pull request by reinaual**
Fixes #2688

### CODE REVIEW

**Comment by reinaual** [comment]
I used the epsilon definitions from [paper](https://doi.org/10.1063/1.469273)

File: src/python/espressomd/electrostatics.pyx [hunk 1]
```diff
@@ -130,6 +130,72 @@ IF ELECTROSTATICS:
                     "r_cut": -1,
                     "check_neutrality": True}
 
+    cdef class RF(ElectrostaticInteraction):
```
Comment (KaiSzuttor) [inline src/python/espressomd/electrostatics.pyx]: Please give it a more descriptive name, e.g. ```ReactionField```

File: src/python/espressomd/interactions.pyx [hunk 2]
```diff
@@ -3304,6 +3304,72 @@ ELSE:
...
+
+    def __init__(self, *args, **kwargs):
+        """
+        QuarticBond initializer. Used to instantiate a QuarticBond identifier
+        with a given set of parameters.
+
+        Parameters
+        ----------
+        k0 : :obj:`float`
+            Specifies the magnitude of the bond interaction.
```
Comment (KaiSzuttor) [inline src/python/espressomd/interactions.pyx]: all parameters describe the same thing? :)

File: testsuite/python/electrostaticInteractions.py [hunk 3]
```diff
@@ -52,7 +52,30 @@ def calc_dh_potential(self, r, df_params):
         u[i] = df_params['prefactor'] * kT * q1 * \
             q2 * np.exp(-df_params['kappa'] * r[i]) / r[i]
         return u
-
+    
+    def calc_rf_potential(self, r, rf_params):
```
Comment (KaiSzuttor) [inline testsuite/python/electrostaticInteractions.py:56]: docstring missing

File: testsuite/python/electrostaticInteractions.py [hunk 4]
```diff
@@ -129,6 +152,53 @@ def test_dh(self):
                                    -f_dh,
                                    atol=1e-2)
         self.system.actors.remove(dh)
+    
+    def test_rf(self):
```
Comment (KaiSzuttor) [inline testsuite/python/electrostaticInteractions.py:158]: docstring

File: testsuite/python/interactions_bonded.py [hunk 5]
```diff
@@ -92,6 +92,24 @@ def test_coulomb(self):
             lambda r: tests_common.coulomb_force(r, coulomb_k, q1, q2),
             lambda r: tests_common.coulomb_potential(r, coulomb_k, q1, q2),
             0.01, self.system.box_l[0] / 3)
+    
+    def test_quartic(self):
```
Comment (KaiSzuttor) [inline testsuite/python/interactions_bonded.py:96]: docstring

**Review by fweik — APPROVED** [review]

**Pull request merged**